libs/ui/src/contest_tally.test.tsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import {
+  CastVoteRecord,
+  Election,
+  ExternalTally,
+  Tally,
+} from '@votingworks/types';
+import { computeTallyWithPrecomputedCategories } from '@votingworks/utils';
+import {
+  electionMultiPartyPrimaryWithDataFiles,
+  electionSample2WithDataFiles,
+} from '@votingworks/fixtures';
+import { render } from '@testing-library/react';
+
+import { ContestTally } from './contest_tally';
+
+function parseCvrsFileContents(cvrsFileContents: string): CastVoteRecord[] {
+  const lines = cvrsFileContents.split('\n');
+  return lines
+    .filter((line) => line.length > 0)
+    .map((line) => JSON.parse(line) as CastVoteRecord);
+}
+
+function constructTally(cvrsFileContents: string, election: Election): Tally {
```

Comment:
Yeah I was thinking about this! At the bare minimum, let me move this `constructTally` function to `@votingworks/test-utils`

Reply:
Changed my mind slightly and moved `parseCvrsFileContents` to `@votingworks/test-utils` while leaving `constructTally` in this file, since `constructTally` is now just a combination of one helper from `@votingworks/test-utils` and another from `@votingwork/utils` 

https://github.com/votingworks/vxsuite/pull/1678/commits/817e02c737fc7282fc9fc17b8085947f18ddf164